src/pages/dashboard/Profile.vue

```diff
@@ -82,16 +82,16 @@
 <script>
 import dashboard from "@/layouts/Dashboard.vue";
 import card from "@/components/Card.vue";
-import GravatarPic from "~/components/GravatarPic.vue";
-import store from "~/store.js";
+// import GravatarPic from "~/components/GravatarPic.vue";
+// import store from "~/store.js";
```

Comment:
This page is unfinished so it's fine to have experimental changes here, not sure what this means though

Reply:
ah, for some reason i think a pre-commit hook was triggering, telling me there was warnings about unused variables and not letting me commit. i forgot but I just commented those out so it would let me commit lol. sorry!

Reply:
Nah, it must be my fault unlinted code got committed in the first place, thanks for this.